engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -69,42 +69,16 @@
  * @author Kevin Pollet &lt;kevin.pollet@serli.com&gt; (C) 2011 SERLI
  * @author Chris Beckey &lt;cbeckey@paypal.com&gt;
  * @author Guillaume Smet
+ * @author Marko Bekhta
  */
 public class ValidatorFactoryImpl implements HibernateValidatorFactory {
```

Comment:
I think I would have kept the changes to `ValidatorImpl` and I wouldn't have included `ValidatorFactoryImpl` and `ValidatorContextImpl` in the change. I would have built the context in the `ValidatorImpl` constructor.